graylog2-server/src/main/java/org/graylog/plugins/sidecar/migrations/V20180212165000_AddDefaultCollectors.java

```diff
@@ -503,18 +510,22 @@ private Optional<Collector> ensureCollector(String collectorName,
                 LOG.debug(msg, collectorName, nodeOperatingSystem);
                 throw new IllegalArgumentException();
             }
-            if (!collector.defaultTemplateUpdated()) {
+
+            if (collector.defaultTemplateUnchanged()) {
+                long oldCRC = collector.defaultTemplateCRC();
```

Comment:
```suggestion
                Long oldCRC = collector.defaultTemplateCRC();
```
This will create a NPE on upgrades from before 5.2. IntelliJ even warns about this!